src/Template/Modules/related.twig

```diff
@@ -1,42 +1,75 @@
-<div class="view-modules">
-
-    {% if error %}
-        <div class="error">{{ error.message }}</div>
-    {% else %}
-
-        {% element 'Toolbar/toolbar_pagination_related' %}
-
-        {# {% element 'Toolbar/toolbar_filters' %} #}
-
-        <div class="list-objects">
-            <nav class="table-header">
-                <div>{{ __('title') }}</div>
-                <div>{{ __('id') }}</div>
-                <div>{{ __('type') }}</div>
-                <div>{{ __('status') }}</div>
-                <div>{{ __('modified')}}</div>
-                {# <div></div> #}
-            </nav>
-
-            {% for object in objects %}
-                <a
-                    href="{{ Url.build({'_name': 'modules:view', 'object_type': object.type, 'id': object.id}) }}"
-                    title="{{ object.attributes.title }}"
-                    class="object-status-{{ object.attributes.status }}">
-                        <div>{{ object.attributes.title }}</div>
-                        <div>{{ object.id }}</div>
-                        <div>{{ object.type }}</div>
-                        <div>{{ object.attributes.status }}</div>
-                        <div>{{ Time.format(object.meta.modified) }}</div>
-                        {# <div>
-                        {% if in_array('DELETE', currentModule.hints.allow) %}
-                            {{ Form.postButton(__('Delete'), {'_name': 'modules:delete', 'object_type': object.type}, {'data': {'id': object.id}})|raw }}
-                        {% endif %}
-                        </div> #}
-                </a>
-            {% else %}
-                {{ __('No items found') }}
-            {% endfor %}
+{% set here=Link.here() %}
+{% set paging = meta.pagination %}
+
+{% element 'Form/_related_script' %}
+
+<div id="related-items_{{ relation }}" class="view-modules">
+
+{% if error %}
+
+    <div class="error">{{ error.message }}</div>
+
+{% else %}
+
+    {# Filter - search related items filter #}
+    <div class="toolbar_filters">
+        {{ Form.control('search-related_' ~ relation, {
+            'id': 'search-related_' ~ relation,
+            'label': __('Filter'),
+            'class': 'search-related',
+            'data-relation-name': relation,
+            'value': Link.query('q')|default(''),
+        })|raw }}
+    </div>
+
+    {# Toolbar pagination #}
+    <div class="toolbar_pagination">
+
+        {# Count of items per module #}
+        <div class="results">
+            {{ paging.count }} {{ currentModule.name }}
```

Comment:
I'd go with `{{paging.count}} items` since generally related object types are different from the current one.